android/src/main/java/com/brentvatne/exoplayer/PictureInPictureUtil.kt

```diff
@@ -74,6 +74,15 @@ object PictureInPictureUtil {
         if (!isSupportPictureInPicture(context)) return
         if (isSupportPictureInPictureAction() && pictureInPictureParams != null) {
             try {
+                if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.TIRAMISU) {
+                    pictureInPictureParams.aspectRatio?.let {
+                        val ratio = it.toFloat()
+                        if (ratio < 0.418410 || ratio > 2.39) {
+                            DebugLog.e(TAG, "Aspect ratio out of range: $ratio. Skipping PiP mode.")
+                            return
+                        }
+                    }
+                }
```

Comment:
@MdAbubakar
I don't think skipping is a good option🤔
In what cases are you getting the ratio out of range?
I'm guessing it's a case that doesn't happen because the logic to set the ratio is in ReactExoplayerView.enterPictureInPictureMode logic, which is currently the only call to this function.